lithops/serverless/backends/aws_lambda/aws_lambda.py

```diff
@@ -629,9 +637,12 @@ def invoke(self, runtime_name, runtime_memory, payload):
         @param payload: invoke dict payload
         @return: invocation ID
         """
-
         function_name = self._format_function_name(runtime_name, runtime_memory)
 
+        if "access_key_id" in payload["config"]["aws"] and "secret_access_key" in payload["config"]["aws"]:
+            del payload["config"]["aws"]["access_key_id"]
+            del payload["config"]["aws"]["secret_access_key"]
```

Comment:
Is it necessary/convenient here to remove the `session_token` too?
I think here you can simply `pop` the keys instead of checking one by one if they exists

```python
payload["config"]["aws"].pop("access_key_id", None)
payload["config"]["aws"].pop("secret_access_key", None)
payload["config"]["aws"].pop("session_token", None)
```